ComplementaryData/modelCuration/Metabolomics_newRxnMetAnnotation.tsv

```diff
@@ -0,0 +1,15 @@
+NewMetName	Charged formula	Charge	compartment	KEGG ID	CHEBI ID	Remark

```

Comment:
could you include an extra column in this table that indicates the DOIs from where each metabolite was detected? As each metabolite should have at least 2, use a blank space to separate them